lisa/environment.py

```diff
@@ -486,15 +486,13 @@ def load_environments(
 class EnvironmentHookSpec:
     @hookspec
     def get_environment_information(self, environment: Environment) -> Dict[str, str]:
-        ...
+        raise NotImplementedError
```

Comment:
It would only raise an error if the spec wasn't fully implemented. In which case, you want it to raise an error.
https://docs.python.org/3/library/exceptions.html#NotImplementedError

Reply:
The spec is optional to implement, it's like a virtual method.